lib/babel-packages.js.flow

```diff
@@ -164,3 +164,9 @@ declare module "@babel/helper-function-name" {
     localBinding?: boolean
   ): any;
 }
+
+declare module "@babel/helper-split-export-declaration" {
+  declare function splitExportDeclaration(exportDeclaration: any): any;
+
+  declare export default typeof splitExportDeclaration;
```

Comment:
Any reason for not using this?
```suggestion
  declare export default function splitExportDeclaration(exportDeclaration: any): any;
```

Reply:
no particular reason, that is what was happend to be generated with flowgen from d.ts